host/volume/zfs/zfs_basics_test.go

```diff
@@ -0,0 +1,142 @@
+package zfs
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-check"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type S struct{}
+
+var _ = Suite(&S{})
+
+func (S) SetUpSuite(c *C) {
+	// Skip all tests in this suite if not running as root.
+	// Many zfs operations require root priviledges.
+	skipIfNotRoot(c)
+}
+
+func (S) TestSnapshotShouldCarryFiles(c *C) {
+	err := WithTmpfileZpool("testpool", func() error {
```

Comment:
Since we do this in every test in this file, make a suite and do the setup in `SetUpTest` and the teardown in `TearDownTest`.